tests/integration/data_finder.yml

```
@@ -422,15 +422,75 @@ get_input_filelist:
       diagnostic: test_diag
       preprocessor: test_preproc
     available_files:
-      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r1i1p1/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r1i1p1.nc
       - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r0i0p0/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r0i0p0.nc
+      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r1i1p1/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r1i1p1.nc
     dirs:
       - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r0i0p0/v20110330/sftlf
     file_patterns:
       - sftlf_fx_HadGEM2-ES_historical_r0i0p0*.nc
     found_files:
       - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r0i0p0/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r0i0p0.nc
 
+  # CMIP6 wildcard specification for fx variable
+
```

```suggestion

  - drs: DKRZ
    variable:
      variable_group: test
      short_name: sftlf
      original_short_name: sftlf
      dataset: HadGEM2-ES
      project: CMIP5
      cmor_table: CMIP5
      institute: [INPE, MOHC]
      frequency: fx
      modeling_realm: [atmos]
      mip: fx
      exp: *
      ensemble: r1i1p1
      diagnostic: test_diag
      preprocessor: test_preproc
    available_files:
      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r1i1p1/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r1i1p1.nc
      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r2i1p1/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r2i1p1.nc
      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r0i0p0/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r0i0p0.nc
    dirs:
      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r0i0p0/v20110330/sftlf
    file_patterns:
      - sftlf_fx_HadGEM2-ES_historical_**.nc
    found_files:
      - MOHC/HadGEM2-ES/historical/fx/atmos/fx/r0i0p0/v20110330/sftlf/sftlf_fx_HadGEM2-ES_historical_r0i0p0.nc
```

To test prioritization of `r0i0p0`.

---

I added the two tests you had suggested. However, I think you meant
```
exp: historical
ensemble: '*'
```
rather than:
```
exp: *
ensemble: r1i1p1
```
So swapped them around.